R/existsBackgroundVariables.r

```diff
@@ -1,4 +1,7 @@
 existsBackgroundVariables <- function(dat, variable, warnIfMissing = FALSE, stopIfMissingOnVars = NULL )  {
+           checkmate::assert_data_frame(dat)
+           checkmate::assert_logical(warnIfMissing, len = 1)
+           checkmate::assert_character(stopIfMissingOnVars, null.ok = TRUE)
            if(is.null(variable))  {return(NULL)}

```

Comment:
I would check "variable" for being a numeric or character vector with null.ok, and then delete row 6 but keep rows 5 and 7

Reply:
I haven't found an easy way to check for 2 specific types at once with checkmate, unfortunately. 
I could only do something similar to line 6 but with different functions (but that wouldn't be an improvement). 
there's the function subset, but it only accepts strings, so you would have to list all different kinds of numeric input, like double, integer, etc. 
assert_subset(typeof(variable), c("character", "numeric", "integer", "double")).

I could add a simple assert_vector(variable), like in #25

Reply:
Oh I just thought about assert_vector() since this accepts character as well as numeric vectors and usage of null.ok is also there...
However, but it is also okay as is

Reply:
like so?